### ISSUE TITLE
Establish branching convention.

### ISSUE THREAD

**Issue by lyle-univ** [TheZimbo16/COS-301-TheInevitables#9]
Currently everything is going into master. At the moment it is not an issue since we don't have any merge conflicts. I suspect when we start coding that will change.

**Comment by lyle-univ** [comment]
The decided upon branching convention is master, wip(work in progress), feature and experimental. Bleeding edge work goes into wip, once the feature is implemented it is merged into feature. Once a working version of the software is developed, it is merged into master and tagged with a version number. All work and ideas that do not fit in the above mentioned branches and conventions can go into experimental.
